libs/brain/tooltip/src/lib/brn-tooltip-trigger.directive.ts

```diff
@@ -627,11 +567,11 @@ export class BrnTooltipTriggerDirective implements OnDestroy, AfterViewInit {
 	}
 
 	/** Updates the tooltip message and repositions the overlay according to the new message length */
-	private _updateTooltipContent() {
+	private _updateTooltipContent(): void {
 		// Must wait for the template to be painted to the tooltip so that the overlay can properly
 		// calculate the correct positioning based on the size of the tek-pate.
 		if (this._tooltipInstance) {
-			this._tooltipInstance.content = this.content;
+			this._tooltipInstance.content = this.brnTooltipTriggerState();
```

Comment:
Same here. And why does it replace this.content?

Reply:
hey, content had an alias which was brnTooltipTrigger, I will refactor the naming